edit-post/assets/stylesheets/_mixins.scss

```diff
@@ -272,6 +272,10 @@
 			margin-left: -18px;
 		}
 	}
+
+	body.is-fullscreen-mode #{$selector} {
+		left: 0 !important;
```

Comment:
I've used important to avoid overriding all the difference use-case above. It just seemed right :)